source/_integrations/fan.modbus.markdown

```diff
@@ -0,0 +1,139 @@
+---
+title: "Modbus Fan"
+description: "Instructions on how to integrate Modbus fans into Home Assistant."
+ha_category:
+  - Fan
+ha_release: 0.108
+ha_iot_class: Local Push
+ha_domain: modbus
+---
+
+The `modbus` fan platform allows you to control [Modbus](http://www.modbus.org/)-enabled fans. We supports two types of Modbus fans — coil-based and register-based. Note that our implementation of Modbus register fan supports only holding registers, because input registers are not writable, and don't give us any control over connected fans.
```

Comment:
```suggestion
The `modbus` fan platform allows you to control [Modbus](http://www.modbus.org/)-enabled fans. We support two types of Modbus fans — coil-based and register-based. Note that our implementation of Modbus register fan supports only holding registers, because input registers are not writable, and don't give us any control over connected fans.
```